patches/chrome-browser-ui-webui-settings-site_settings_helper.cc.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/browser/ui/webui/settings/site_settings_helper.cc b/chrome/browser/ui/webui/settings/site_settings_helper.cc
-index cc79a7320b12986756dfeed5cd9f3283373dd6f4..b4ed1441c1755eed3863ab28950ba3cbb0810d01 100644
+index 0f5493f32679e17705f9b5d12346e83a36acfe01..dbc7e70271f9069a5f53c64678bbe934b77451b9 100644
 --- a/chrome/browser/ui/webui/settings/site_settings_helper.cc
 +++ b/chrome/browser/ui/webui/settings/site_settings_helper.cc
-@@ -375,6 +375,7 @@ bool HasRegisteredGroupName(ContentSettingsType type) {
+@@ -149,6 +149,7 @@ const ContentSettingsTypeNameEntry kContentSettingsTypeGroupNames[] = {
+     {ContentSettingsType::INSECURE_PRIVATE_NETWORK, nullptr},
+     {ContentSettingsType::PERMISSION_AUTOREVOCATION_DATA, nullptr},
+     {ContentSettingsType::FILE_SYSTEM_LAST_PICKED_DIRECTORY, nullptr},
++    BRAVE_CONTENT_SETTINGS_TYPE_GROUP_NAMES_LIST
```

Comment:
same as https://github.com/brave/brave-core/pull/7457/files#r549508112 except we'll have to override ContentSettingsTypesFromGroupNames instead of ContentSettingsTypeFromGroupName

Reply:
actually is this even needed?